docs/designers-developers/developers/packages.md

```diff
@@ -22,7 +22,7 @@ const { PlainText } = wp.editor;
 
 ```
 
-## Using the packages via npm
+## Using the Packages via NPM
```

Comment:
```suggestion
## Using the Packages via npm
```

"npm" should [always be lower-case](https://css-tricks.com/start-sentence-npm/). (`lowercase_npm_dangit()`)

Reply:
(As a note typically style guides do often overrule how brands prefer their names capitalized, but given our own `capital_P_dangit()` history I think it's a nice sign of respect to use lower-case here, as they desire!)